routes/api.php

```diff
@@ -27,6 +30,11 @@
     Route::resource('tag', TagController::class)->except(['create', 'edit']);
     Route::resource('sender-identity', SenderIdentityController::class)->except(['create', 'edit']);
     Route::resource('campaign', CampaignController::class)->except(['create', 'edit']);
+    Route::resource('role', RoleController::class)->except(['create', 'edit']);
+    Route::resource('permission', PermissionController::class)->except(['create', 'edit']);
+    Route::resource('user/user', UserController::class)->except(['create', 'edit']);
```

Comment:
why double `user/user`?

Reply:
Because the `user` route is already defined in line number 24 which returns the details of the login user after the user is logged in and after the user is authenticated. And in this `user/user` route we are fetching the details of all the users or can edit or delete any user. And if I keep both the routes same then it automatically hits the wrong route and instead of returning the details of the login user it returns all the users. That's why I have named both routes separately to keep both routes separate.
Please let me know, your thought?

Reply:
I think we can rename first API (line 24) as:
```php
Route::middleware(['auth:sanctum'])->get('/profile', function (Request $request) {
    return $request->user();
});
```

Or something similar. And rename UserController route to just `/user`.

Also, can we keep one middleware grouping and move the `profile` route inside the same group?